vsphere/internal/helper/structure/structure_helper.go

```diff
@@ -626,3 +626,30 @@ func ValuesAvailable(base string, keys []string, d *schema.ResourceDiff) bool {
 	}
 	return true
 }
+
+func DiffSlice(a, b []interface{}) []interface{} {
+	var c []interface{}
+	for _, aa := range a {
+		found := false
+		for _, bb := range b {
+			if reflect.DeepEqual(aa, bb) {
+				found = true
```

Comment:
should we break here?

Reply:
In this one we still need to run through all the remaining possibilities because we're looking for ones that do not have a match.